sympy/printing/julia.py

```diff
@@ -418,7 +418,7 @@ def _print_yn(self, expr):
 
 
     def _print_Piecewise(self, expr):
-        if expr.args[-1].cond != True:
+        if len(expr.args) == 0 or expr.args[-1].cond != True:
```

Comment:
I thought all of these codegen were working correctly before?

Reply:
I changed this because the codegens were raising a tuple index out of range error later on in the function instead of raising the relevant ValueError that there isn't a default block.